scylla-server/src/services/systems.services.ts

```diff
@@ -0,0 +1,15 @@
+import prisma from '../prisma/prisma-client';
+import { ResponseFunction } from '../utils/message-maps.utils';
+
+// CRUD operation to get all systems with ResponseFunction type
+
+export const getAllSystems: ResponseFunction = async (data: JSON) => {
```

Comment:
since were not using data in this particular function (its undefined) you can preface it like this _data so that the linting thing goes away